lib/Business/CyberSource/RequestPart/Item.pm

```diff
@@ -76,6 +76,14 @@ has national_tax => (
 	is          => 'rw',
 );
 
+has invoice_number => (
+    isa => NonEmptySimpleStr,
+    remote_name => 'invoiceNumber',
+    traits => ['SetOnce'],
```

Comment:
Either remove `['SetOnce']` or make `is => 'rw'` ; `ro` takes precedence so leaving it this way is confusing. My preference is to remove `['SetOnce']`.